src/libsync/owncloudpropagator.cpp

```diff
@@ -323,6 +323,27 @@ bool PropagateItemJob::checkForProblemsWithShared(int httpStatusCode, const QStr
     return false;
 }
 
+void PropagateItemJob::readFileIdHeaders(const QNetworkReply &reply)
```

Comment:
Ah, I did this automatically because it can't be 0. But yes, it's not the style we use and will adjust it.